apps/frontend/src/app/OrderElement/OrderElement.tsx

```diff
@@ -0,0 +1,73 @@
+import { Grid, styled, TextField, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import { OrderModel, OrderStatus } from '@pasnik/api/data-transfer';
+import { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+
+export default function OrderElement(props: {
+  order: OrderModel;
+  index: number;
+}) {
+  const { order, index } = props;
+  const Tile = styled(Box)(({ theme }) => ({
```

Comment:
this should be extracted from the component itself as it does not depend on any components data same goes for `StyledTextField`